Bio/SeqUtils/__init__.py

```diff
@@ -11,42 +11,111 @@
 
 
 import re
+import warnings
+from collections import Counter
 from math import pi, sin, cos
 
 from Bio.Seq import Seq
 from Bio.Data import IUPACData
-
+from Bio import BiopythonDeprecationWarning
 
 ######################################
 # DNA
 ######################
 # {{{
 
+# Nucleotides not present here are treated as having value 0.
+ambiguous_gc_values = {
```

Comment:
Do you want to leave this dictionary public (as it is), or add a leading underscore to imply it is private?